src/details/ArborX_DetailsBoundingVolumeHierarchyImpl.hpp

```diff
@@ -386,13 +384,12 @@ void BoundingVolumeHierarchyImpl<DeviceType>::queryDispatch(
   // [ 0 2 4 .... 2N-2 2N ]
   //                    ^
   //                    N
-  exclusivePrefixSum(offset);
-
+  //
   // Let us extract the last element in the view which is the total count of
   // objects which where found to meet the query predicates:
```

Comment:
May need to teak this comment

Reply:
Don't both of them still apply?

Reply:
OK. I thought the combination sounded weird, but maybe not.